src/main/java/com/codingnomads/betty/logic/services/ProbabilityToWinService.java

```diff
@@ -0,0 +1,44 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.models.TeamProbabilityToWin;
+import com.codingnomads.betty.logic.models.TeamSentimentScore;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProbabilityToWinService {
+
+    public TeamProbabilityToWin getProbabilityToWinFromSentimentAnalysis(TeamSentimentScore teamSentimentScore) {
+        if (teamSentimentScore.getScore() == null) {
+            throw new NullPointerException("Team Score is null");
```

Comment:
Try to always create your own Exceptions explaining the situation that is happening. For example a InvalidScoreException

Reply:
![tenor-68868280](https://user-images.githubusercontent.com/32600398/56576593-6065ed00-65fb-11e9-8e33-4e55a6c94a78.gif)